shardingsphere-infra/shardingsphere-infra-binder/src/main/java/org/apache/shardingsphere/infra/binder/segment/select/orderby/engine/OrderByContextEngine.java

```diff
@@ -78,4 +96,42 @@ private OrderByContext createOrderByContextForDistinctRowWithoutGroupBy(final Se
         }
         return null;
     }
+    
+    private OrderByContext createOrderByContextForMySQLSelectWithoutOrderBy(final ShardingSphereSchema schema, final SelectStatement selectStatement, final GroupByContext groupByContext) {
+        if (groupByContext.getItems().size() > 0) {
```

Comment:
`groupByContext.getItems().size() > 0` can be replace to `!groupByContext.getItems().isEmpty()`